src/component/MovieTile/MovieTile.js

```diff
@@ -0,0 +1,29 @@
+import './MovieTile.css';
+import MenuButton from '../MenuButton/MenuButton';
+
+function MovieTile({ imageUrl, movieName, releaseYear, genres, onSelect, id}) {
+  console.log("Started rendering movie tile component");
+  const commaSeparatedList = genres.join(', ');
+  
+  function handleOnClick() {
```

Comment:
Suggestion: can use arrow function here , `const handleOnClick = () => onSelect(id)`

Reply:
Fixed as part of 5ea7447